app/models/models_elocaps.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
```

Comment:
Is this needed?

Reply:
Yes, because there are references to the `GamePlayer` type in the type annotations, and it can't resolve them without this import.